datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/type/AspectInterfaceTypeResolver.java

```diff
@@ -0,0 +1,20 @@
+package com.linkedin.datahub.graphql.resolvers.type;
+
+import graphql.TypeResolutionEnvironment;
+import graphql.schema.GraphQLObjectType;
+import graphql.schema.TypeResolver;
+
+/**
+ * Responsible for resolving the {@link com.linkedin.datahub.graphql.generated.Aspect} interface type.
+ */
+public class AspectInterfaceTypeResolver implements TypeResolver {
+
+    public AspectInterfaceTypeResolver() { }
+    @Override
+    public GraphQLObjectType getType(TypeResolutionEnvironment env) {
+      // TODO(Gabe): Fill this out. This method is not called today. We will need to fill this
+      // out in the case we ever want to return fields of type Aspect in graphql. Right now
+      // we just use Aspect to define the shared `version` field.
+      return null;
```

Comment:
Can we fit this out with something in this PR to set the stage for future extension?

Reply:
I'm not sure- there are a few ways this could be implemented. Since this method is never called at the moment, I'd rather wait to implement it when we are more clear on requirements.